homeassistant/components/group/light.py

```diff
@@ -189,6 +192,24 @@ def should_poll(self) -> bool:
         if ATTR_COLOR_TEMP in kwargs:
             data[ATTR_COLOR_TEMP] = kwargs[ATTR_COLOR_TEMP]
 
+            # Create a new entity list to mutate
+            updated_entities = list(self._entity_ids)
+
+            # Walk through initial entity ids, split entity lists by support
+            for entity_id in self._entity_ids:
+                state = self.hass.states.get(entity_id)
+                if not state:
+                    continue
+                else:
```

Comment:
Please remove `else:` and outdent below since we `continue` above.

Reply:
Done. @MartinHjelmare